packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html

```diff
@@ -389,6 +389,107 @@ <h2>
+                        data-ng-repeat="sshKey in $ctrl.addedSshKeys track by $index"
+                    >
+                        <fieldset class="oui-inline-adder__fieldset">
+                            <legend
+                                data-ng-bind=":: 'pci_projects_project_training_jobs_submit_select_ssh_legend' | translate"
+                            ></legend>
+                            <div class="oui-inline-adder__row">
+                                <div class="oui-inline-adder__field">
+                                    <oui-field
+                                        data-size="m"
+                                        data-label="{{:: 'pci_projects_project_training_jobs_submit_select_ssh_name_tips' | translate  }}"
+                                        data-ng-if="$ctrl.allKeyNames.length > 0"
+                                    >
+                                        <oui-select
+                                            id="savedKeyName"
+                                            name="savedKeyName"
+                                            data-model="$ctrl.addedSshKeys[$index].keyName"
+                                            data-items="$ctrl.allKeyNames"
+                                            on-change="$ctrl.changeSavedSshKey($index)"
+                                        >
+                                        </oui-select>
+                                    </oui-field>
+                                    <oui-field
+                                        data-label="{{:: 'pci_projects_project_training_jobs_submit_select_ssh_input_tips' | translate  }}"
+                                    >
+                                        <textarea
+                                            type="textarea"
+                                            class="oui-input"
+                                            placeholder="{{:: $ctrl.JOB_SSH_KEYS_CONSTANTS.PLACEHOLDER }}"
+                                            name="sshPublicKey"
+                                            rows="{{:: $ctrl.JOB_SSH_KEYS_CONSTANTS.NB_ROWS }}"
+                                            data-ng-model="$ctrl.addedSshKeys[$index].model"
+                                            data-ng-required
+                                            data-ng-disabled="$ctrl.addedSshKeys[$index].disabled"
+                                        />
+                                    </oui-field>
+                                </div>
+                            </div>
+                        </fieldset>
+                        <footer class="oui-inline-adder__footer">
+                            <button
+                                type="button"
+                                class="oui-inline-adder__action oui-button oui-button_primary oui-button_s"
+                                data-translate-attr="{title: 'dedicated_server_install_image_config_http_header_delete'}"
```

Comment:
**issue(blocking):** Looks like this translation is coming from a copy/paste. We should used a scoped translation.